portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,43 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+/* Servlet that returns json of commentaries statistics*/
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
+  /* Some hardcoded examples*/
+  String [] commentList = {"Hi! I'm Mark", "Hello, my name is Nancy :3", "Good morning/afternoon/evening! I'm Alex"};
+  String [] nameList = {"Mark", "Nancy", "Alex"};
+  String [] ratingList = {"4/5", "5/5", "0/5"};
+  int listLength = commentList.length;
+
+  /* Processing of GET request to /comment page, json dtring will be returned*/
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    String json = "{";
+    json += convertToJsonList("comment", commentList);
+    json += ",";
+    json += convertToJsonList("name", nameList);
+    json += ",";
+    json += convertToJsonList("rating", ratingList);
+    json += "}";
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
```

Comment:
"application/json;" is likey to be reused multiple times, so it might be good to extract it as a constant somewhere in a shared class in future when you add more servlets that return JSON content.